### DISCUSSION TITLE
No Save button displayed in Add Table panel

### DISCUSSION THREAD

**Discussion by drrwebber** [supabase/supabase#17497] [Contribute to Supabase]
Running Chrome or Edge on Android tablet - cannot see the 'Save' button anywhere.
Probably a newbee dumb answer - but I've tried all the window sizing / moving tricks I know.
See screenshot.
![Screenshot_20230916_153016_Chrome](https://github.com/supabase/supabase/assets/1458927/8ee83ef7-10cf-4999-9d71-866c9540ad80)

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/36e4c212-9934-4125-a5b2-40ed7687c0ad)
Chrome.
Try a hard refresh of the browser.

**Reply by drrwebber** [reply]
Gary - thanks - that prompted me to retry Edge - see below - figured out the resolution - not good - looks like a UX bug.

**Comment by drrwebber** [comment]
Oh my - I figured this out - its the scrollbars in the pane box that are not working correctly.
I have a 'virgin' copy of Edge installed -  and I fiddled around between full screen and not. And slide the scrollbar sliders to the left and right - eventually the Save button "peeped' out of the bottom corner. This looks like a UX bug in the JS rendering library. YIKES!
Screenshot of Edge.
![Screenshot_20230916_174213_Edge](https://github.com/supabase/supabase/assets/1458927/d4f4b42e-9e0e-4e4c-8ad5-f60d8a02fe39)
